src/euphorie/client/browser/organisation.py

```diff
@@ -129,9 +129,14 @@ def organisations(self):
             return []
 
         organisations = []
-        if account.organisation:
-            # Add the user own organisation (if present)
-            organisations.append(account.organisation)
+        if not account.organisation:
```

Comment:
Is this a write on read? Doesn't plone.protect prevent yopu to do this?

Reply:
It should bark anyway, see https://github.com/EU-OSHA/oira.application.buildout/pull/8